make/utils_exposed.cmake

```diff
@@ -170,16 +170,30 @@ function (DynamoRIO_get_full_path out target loc_suffix)
       set(suffix ${CMAKE_${target_type}_SUFFIX})
     endif ()
     set(output_dir "")
-    if (target_type STREQUAL "SHARED_LIBRARY" OR target_type STREQUAL "MODULE_LIBRARY")
-      get_target_property(library_dir ${target} LIBRARY_OUTPUT_DIRECTORY${loc_suffix})
-      set(output_dir ${library_dir})
-    elseif (target_type STREQUAL "STATIC_LIBRARY")
-      get_target_property(archive_dir ${target} ARCHIVE_OUTPUT_DIRECTORY${loc_suffix})
-      set(output_dir ${archive_dir})
-    elseif (target_type STREQUAL "EXECUTABLE")
-      get_target_property(runtime_dir ${target} RUNTIME_OUTPUT_DIRECTORY${loc_suffix})
-      set(output_dir ${runtime_dir})
+    if (WIN32)
+      if (target_type STREQUAL "MODULE_LIBRARY")
+        get_target_property(library_dir ${target} LIBRARY_OUTPUT_DIRECTORY${loc_suffix})
+        set(output_dir ${library_dir})
+      elseif (target_type STREQUAL "STATIC_LIBRARY")
+        get_target_property(archive_dir ${target} ARCHIVE_OUTPUT_DIRECTORY${loc_suffix})
+        set(output_dir ${archive_dir})
+      elseif (target_type STREQUAL "EXECUTABLE" OR target_type STREQUAL "SHARED_LIBRARY")
+        get_target_property(runtime_dir ${target} RUNTIME_OUTPUT_DIRECTORY${loc_suffix})
+        set(output_dir ${runtime_dir})
+      endif ()
+    else ()
+      if (target_type STREQUAL "SHARED_LIBRARY" OR target_type STREQUAL "MODULE_LIBRARY")
+        get_target_property(library_dir ${target} LIBRARY_OUTPUT_DIRECTORY${loc_suffix})
+        set(output_dir ${library_dir})
+      elseif (target_type STREQUAL "STATIC_LIBRARY")
+        get_target_property(archive_dir ${target} ARCHIVE_OUTPUT_DIRECTORY${loc_suffix})
+        set(output_dir ${archive_dir})
+      elseif (target_type STREQUAL "EXECUTABLE")
+        get_target_property(runtime_dir ${target} RUNTIME_OUTPUT_DIRECTORY${loc_suffix})
+        set(output_dir ${runtime_dir})
+      endif ()
     endif ()
+    string(REPLACE "/./" "/" output_dir ${output_dir})
```

Comment:
There's no comment: unclear why this is needed?

Reply:
convert_to_NT_file_path can't handle it. The alternative would be to fix/change the former, but I decided it's better to replace it here. What do you think

Reply:
Please comment such unusual things.  But I don't understand why this would show up on Windows: the native Windows cmake would use backslashes.  Were you using the cygwin cmake?  I didn't think we supported that.

Reply:
Removed (apparently cygwin/cmake is creating unix style paths, Windows is fine, line not needed)